pkg/storage/tree/pprof.go

```diff
@@ -67,12 +67,14 @@ var DefaultSampleTypeMapping = map[string]*SampleTypeConfig{
 		Sampled:     true,
 	},
 	"objects": {
-		Units:      "objects",
```

Comment:
Messaged in slack, but basically, I'm trying to figure out if we need `objects` and `space` at all. It seems like these are very similar to `inuse_objects` and `inuse_space`

Reply:
Makes sense. 

On latest `@pyroscope/nodejs` I adjust profile to meet golang sample types name

https://github.com/pyroscope-io/pyroscope-nodejs/pull/12